### ISSUE TITLE
Feature request - Open in a new tab for the players

### ISSUE THREAD

**Issue by DarKDinDoN** [EvanesceExotica/Journal-To-Canvas-Slideshow#3]
Hello,

This module is awesome!!!

Would it be possible to have the images & videos to open in a new popup rather than in a scene? Maybe as an option?
As a GM, I would like to keep the background of my scene (map, ambiance, battlemap, etc.) and have a popup above the visible scene. And with your module, it would make this process a lot easier 😃

Example:

- Characters are travelling across the map displayed as a scene
- GM describes the mountains in front of them and show a picture of the mountains
- GM then proceeds to describe the fortress above them and change the image accordingly
- etc.

But all in one popup from one journal note, just like what your module does. So the players don't have to close a lot of popups and the GM doesn't have to create a million journal notes.

Thx!

**Comment by EvanesceExotica** [comment]
Thank you! This is a really cool idea, and definitely something I can try to do for the next update. ^ ^ What I'm thinking might work is to have a separate, dedicated journal entry in Image Mode that the players can have open too, and try and change the image of that to whatever image you click in the original journal entry. 

Would a popup like this work?

![image](https://user-images.githubusercontent.com/13098820/103367778-f1a98a00-4a93-11eb-8aa3-40c6bcb9476f.png)

Right now I'm also figuring out how to create custom configuration options for this module, so I can definitely make that an option too (showing the image in the Display scene vs in a popup window) once I get it all worked out.

**Comment by DarKDinDoN** [comment]
For settings, that's rather simple. You register a setting for you module namespace:

Example:
```javascript
game.settings.register(constants.moduleName, "settingName", {
    name: "Name displayed",
    hint: "Hint displayed",
    scope: "client",
    config: true,
    default: false,
    type: Boolean,
    onChange: value => {
      callback();
    }
  });
```


As for the popup, that would be great. But you don't need for the GM to create a decicated journal entry.
Take a look at this module: https://github.com/zeel01/TokenHUDArtButton

You can open a popout without having a journal entry created. You can even immediately share it to all players.
Once this is done, you can either close this one and reopen a new one (the easy way I think), or try to change the content.

**Comment by DarKDinDoN** [comment]
@EvanesceExotica Created a PR for that https://github.com/EvanesceExotica/Journal-To-Canvas-Slideshow/pull/4

**Comment by EvanesceExotica** [comment]
Hey there, so sorry for this taking so long and my replies being sparse! Just started a new semester in school and have been a bit swamped haha. 

I just updated the module; it now has settings so that you can switch back and forth between the journal images opening in a scene and opening in a window, however I am using the journal entry method I mentioned as I unfortunately had finished working on it before I saw your proposed solution :(

Hopefully that will work for now! But I'll approve your pull request too! ^ ^ 

Thanks for the suggestion, it was an excellent one!

**Comment by EvanesceExotica** [comment]
This feature was implemented in v0.1.4. ^ ^
